products/compute/api.yaml

```diff
@@ -3293,6 +3293,15 @@ objects:
         update_verb: :POST
         update_url:
           'projects/{{project}}/regions/{{region}}/forwardingRules/{{name}}/setTarget'
+      - !ruby/object:Api::Type::Boolean
+        name: 'allowGlobalAccess'
+        min_version: beta
+        description: |
+          If true, clients can access ILB from all regions.
+          Otherwise only allows from the local region the ILB is located at.
+        send_empty_value: true
+        update_verb: :PATCH
+        update_url:  projects/{{project}}/regions/{{region}}/forwardingRules/{{name}}
```

Comment:
For https://github.com/GoogleCloudPlatform/magic-modules/pull/2400 we had this as `min_version: beta`. Checking  the [GA API](https://cloud.google.com/compute/docs/reference/rest/v1/forwardingRules/insert) I don't see `allowGlobalAccess` included there yet, as it is for the [beta version](https://cloud.google.com/compute/docs/reference/rest/beta/forwardingRules/insert) so we may want to limit this for TPGB for now.

Reply:
Ignore this. I see it there now. :)